### DISCUSSION TITLE
dipy vis add a figure title

### DISCUSSION THREAD

**Discussion by ucigrangers** [dipy/dipy#2657] [Scientific Visualization]
Hello, I am working on adding titles to figures with actor.text_3d. 

For some reason the text renders in an oblique way with the following parameters:     panel1 = actor.text_3d(str(i),position=(0,0,0),justification='left',vertical_justification='bottom')

Is there a way to add 2-dimensional text to an image with dipy viz? I am hoping to avoid using PIL for this but I would just like to label the image directly above or below in a horizontal fashion rather than oblique. 
![sub-01_Uncinate_L trk](https://user-images.githubusercontent.com/29385931/194915011-97f2bab1-5f6a-47e0-8ef1-58f881535e2c.png)

**Comment by skoudoro** [comment]
Hi @ucigrangers,

you need to set up your text to follow the camera. this is currently not available. it should be added in the next release. 
So you have 3 choices:

1.  implement it (text actor to follow the camera)
2. use master branch of FURY (instead of 0.8.0) since it is implemented as you can see on this PR https://github.com/fury-gl/fury/pull/661
3. use another method like using the grid actor (demo here: https://fury.gl/latest/auto_examples/viz_animated_surfaces.html#sphx-glr-auto-examples-viz-animated-surfaces-py)


more info on this thread: https://github.com/fury-gl/fury/discussions/658

I hope it helps
